pkg/internal/apis/acme/types_issuer.go

```diff
@@ -170,6 +170,7 @@ type ACMEChallengeSolverHTTP01Ingress struct {
 
 	// The ingress class to use when creating Ingress resources to solve ACME
 	// challenges that use this challenge solver.
+	// A reference by name to a Kubernetes [IngressClass](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class) object
```

Comment:
I prefer not to have markdown here, while it is nice for the site this is also shown for `kubectl explain` where this will look weird

Reply:
Ah yes makes sense.

Would `a Kubernetes IngressClass (see docs at x)` be a better solution?

Reply:
Yup! sounds good!